### ISSUE TITLE
References are style inconsistently (with/without first name)

### ISSUE THREAD

**Issue by jeroenh** [CERTCC/SSVC#237]
It seems that the default style of pandoc is not completely consistent, and not reproducibly so.

The current setup results in unpredictable results, as can be seen in the section `Current state of practice`.

[@spring2018litrev] -> “Review of Human Decision-Making During Computer Security Incident Analysis” 2021
[@cvss_sig] -> Wiles and Dugal 2019
[@farris2018vulcon] -> Farris et al. 2018
[@spring2018cvss] -> Jonathan M. Spring et al. 2018

The second and third seem to be using APA like references, but the first and last don't.
Using the current Makefile, I also can't reproduce this result, as the first reference on my system results in "Jonathan M. Spring and Illari 2021", but the others are the same.

My suggestion would be to pick a `csl` style file from https://citationstyles.org and add it to the repository, and use the `--csl` option of pandoc to reference that file (with proper attribution as per the license).

**Comment by j---** [comment]
At least some of these are errors in the bib file. For example:
https://github.com/CERTCC/SSVC/blob/eb94a2b50ec2eb26067ebccb86bf9b598b4d68e4/doc/md_src_files/sources_ssvc.bib#LL58-L58
Is wrong, it should assign the author variable. 

``author = {Jonathan M Spring and Phyllis Illari}``

Woops. 
I wouldn't be surprised if the difference between how [@farris2018vulcon] and [@spring2018cvss]  are styled in the text is a difference in the way that articles and techreports are handled by the style file. There should be a setting somewhere to control how many authors will be listed explicitly before switching to "et al." in text, but 2 or 3 are really the only viable options, and both of the et al. stylings cited here have 4+ authors, so that seems normal.  

The 2018 tech report is content similar to the 2021 IEEE article, we could also solve this by replacing all the references to the tech report, if that is easier than sorting out styles. 

@article{spring2021time,
  title={Time to Change the CVSS?},
  author={Spring, Jonathan and Hatleback, Eric and Householder, Allen and Manion, Art and Shick, Deana},
  journal={IEEE Security \& Privacy},
  volume={19},
  number={2},
  pages={74--78},
  year={2021},
  publisher={IEEE}
}

However, specifying a CSL file seems like a good idea. Pandoc is already grabbing one by default:

> These files are specified using the [--csl](https://pandoc.org/MANUAL.html#option--csl) option or the csl (or citation-style) metadata field. By default, pandoc will use the [Chicago Manual of Style](https://chicagomanualofstyle.org/) author-date format. 

https://pandoc.org/MANUAL.html#specifying-a-citation-style

I would prefer to avoid a citation style war. Chicago is OK. Might be good to at least formalize that as the default by specifying it in https://github.com/CERTCC/SSVC/blob/eb94a2b50ec2eb26067ebccb86bf9b598b4d68e4/doc/Makefile#LL72

But that makes me think these issues are more with the bibliography file than the style.

**Comment by j---** [comment]
@jeroenh , thoughts on whether issue is with bib file rather than style?

**Comment by jeroenh** [comment]
This issue is not with the bib file, and I have no idea what is causing it.
I just tested by changing your last-name in the bib-file by either adding, removing or changing a single letter causes it to change style, i.e. then it does not include the first name.

**Comment by jeroenh** [comment]
I wonder if this is because of the declaration of your authorship in the `pdf-styling.yaml` file.

**Comment by jeroenh** [comment]
Nope that is not it either, I changed the name in the pdf-styling.yaml file and it did not result in any changes even after `make clean`.

This bug keeps getting weirder and weirder.
Changing the first name part in the `.bib` file does not cause any changes in style. Changing the initial to a different letter results in the first name being abbreviated (which is still inconsistent with the rest of the styling).

I have also tried changing the type of the entry, and that does not make a difference either.
